Model/Importer.php

```diff
@@ -81,6 +81,7 @@ private function validateData(array $dataArray): void
      */
     private function importData()
     {
+        $this->resetImportModel();
```

Comment:
Maybe worth adding a comment why this is necessary? At least for me, this is not obvious :)